website/docusaurus.config-blog.js

```diff
@@ -50,6 +50,26 @@ const config = {
 
           blogSidebarTitle: 'All posts',
           blogSidebarCount: 'ALL',
+          feedOptions: {
+            type: 'rss', // or whichever feed types you prefer
+            title: 'The FerretDB Blog',
+            description: 'Welcome to the FerretDB blog- The truly open source replacement for MongoDB',
+            copyright: `Copyright © ${new Date().getFullYear()} FerretDB, Inc.`,
```

Comment:
It sounds strange to add the current year to previous year blog posts

Reply:
This just describes the copyright for the feeds, not necessarily the date for the blog posts. We have it on the blog and website.
<img width="369" alt="Screenshot 2023-09-22 at 14 38 09" src="https://github.com/FerretDB/FerretDB/assets/24963785/ae1e9460-c5f3-45ce-a30b-44f3bae395c3">
site